### PR TITLE
add notes about migrating to hackerone

### PR DESCRIPTION
**Pull request by fkiriakos07**
## One-line summary

Add notes about migrating web bug bounty to hackerone

## Screenshots

Main page
Screenshot 2023-05-05 at 11 32 36

Screenshot 2023-05-05 at 11 18 24

image

Bounty table
image

### CODE REVIEW

File: bedrock/security/templates/security/bug-bounty/web-hall-of-fame.html [hunk 1]
```diff
@@ -21,6 +21,7 @@ <h1 class="mzp-c-article-title">Web And Services Bug Bounty Hall of Fame</h1>
 
   <p>On behalf of the Mozilla and the millions of people who visit our sites, use Firefox and our other products we would like to thank them for their hard work in helping to make us more secure.</p>
   <p>As of this date, we have paid out over <b>$4,000,000</b> across all of our bounties. Congratulations to everybody who has participated!</p>
-  {% include "security/partials/hall-of-famers-list.html" %}
   <p>If your name is on the list incorrectly or you feel you should be on the list please feel free to mail us at <a href="mailto:security@mozilla.org">security@mozilla.org</a>.</p>
+  <p><strong>Our program has been migrated to HackerOne.</strong> Therefore, we will discontinue adding researchers on this page. Starting from May 2023, researchers will be recognized on our HackerOne pages: <a href="https://hackerone.com/mozilla_critical_services">Mozilla Critical Services</a> and <a href="https://hackerone.com/mozilla_core_services">Mozilla Core Services</a></p>
```
Comment (gene1wood) [inline bedrock/security/templates/security/bug-bounty/web-hall-of-fame.html:25]:
```suggestion
  <p><strong>Our Bug Bounty Program has been migrated to HackerOne.</strong> Therefore, we will discontinue adding researchers on this page. Starting from May 2023, researchers will be recognized on our HackerOne pages, <a href="https://hackerone.com/mozilla_critical_services">Mozilla Critical Services</a> and <a href="https://hackerone.com/mozilla_core_services">Mozilla Core Services</a></p>
```

**Review by gene1wood — CHANGES_REQUESTED** [review]

File: bedrock/security/templates/security/bug-bounty/faq-webapp.html [hunk 2]
```diff
@@ -77,11 +77,9 @@ <h2 id="eligible-bugs">Eligible bugs</h2>
   <dl class="mzp-u-list-styled">
     <dt id="eligible-bugs">Which domains and web applications will be considered to be part of the bug bounty?</dt>
     <dd>
-      <p>Although all of our non-community websites are considered in scope for acknowledgment on our
-      <a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">hall of fame</a>, only
-      a limited subset are eligible for monetary payouts. Please see our
+      <p>Only a limited subset of our websites are eligible for monetary payouts. Please see our
       <a href="{{ url('security.bug-bounty.web-eligible-sites') }}">list of eligible sites</a>
-      for additional information.</p>
+      for additional information. Note that some low severity issues are not eligible for monetary awards based on their impact. We will recognize the reporter by thanking them on our HackerOne pages.</p>
```
[the remark this answers is not in the document]
Comment (fkiriakos07) [reply bedrock/security/templates/security/bug-bounty/faq-webapp.html:82]: HackerOne don't have a hall of fame page, each program has a Hacktivity section where reporters are recognized

[the remark this answers is not in the document]
Comment (fkiriakos07) [reply bedrock/security/templates/security/bug-bounty/faq-webapp.html:82]: actually on the Thanks page: https://hackerone.com/mozilla_core_services/thanks

[the remark this answers is not in the document]
Comment (gene1wood) [reply bedrock/security/templates/security/bug-bounty/faq-webapp.html]: Cool, that change looks good.

**Review by gene1wood — APPROVED** [review]

**Comment by fkiriakos07** [comment]
Hello @stevejalim can you please take a look at this PR?

**Comment by stevejalim** [comment]
@craigcook @stephaniehobson either of you got a mo to review this, please? Am OOO now

**Review by craigcook — APPROVED** [review]
🐞

**Pull request merged**